src/encoded/tests/test_schema_target.py

```diff
@@ -187,3 +195,26 @@ def test_target_organism_uuid(testapp, human):
     }
     res = testapp.post_json('/target', item).json['@graph'][0]
     assert res['organism'] == human['@id']
+
+
+def test_synthetic_tag_target(testapp, target_synthetic_tag, human, ctcf):
+    flag_target = testapp.post_json(
+        '/target', target_synthetic_tag
+    ).json['@graph'][0]
+    assert flag_target['title'] == 'FLAG (Synthetic tag)'
+
+    syn_and_org = {'target_organism': human['uuid']}
+    syn_and_org.update(target_synthetic_tag)
+    res = testapp.post_json('/target', syn_and_org, expect_errors=True)
+    assert res.json['status'] == 'error'
+    syn_and_org['investigated_as'] = ['synthetic tag', 'tag']
```

Comment:
This just tests if synthetic tag still triggers dependency issue if part of a group?

Reply:
Correct.